src/Blocks/AbstractBlocks.php

```diff
@@ -131,7 +131,7 @@ function ($block) {
 		$allowedBlockTypes[] = 'core/block';
 		$allowedBlockTypes[] = 'core/template';
 
-		return $allowedBlockTypes;
+		return \apply_filters('all_allowed_block_list', $allowedBlockTypes, $blockEditorContext);
```

Comment:
Suggestion:
```suggestion
		return \apply_filters('es_allowed_blocks', $allowedBlockTypes, $blockEditorContext);
```
Just so it's not _that_ easy to confuse it with `allowed_block_types_all`